bin.ts

```diff
@@ -1,13 +1,24 @@
 #!/usr/bin/env node
 
 import { QuickImportCycleChecker } from './src/quick-import-cycle-checker';
+import { parseArgs } from './src/util/parse-arguments.util';
 
-// TODO Make this more elegant/configurable
-const directoryToCheck = process.cwd();
-const rootDirectory = process.cwd();
+export interface cliSettings {
+    checkDirectories: string[];
+    root: string;
+    exclusions: RegExp[];
+}
 
-QuickImportCycleChecker.forDirectories(directoryToCheck)
-    .withRootDirectory(rootDirectory)
+const argumentsParsed = parseArgs(process.argv);
+const settings: cliSettings = {
+    checkDirectories: argumentsParsed['--checkDirs'] ?? [process.cwd()],
+    root: argumentsParsed['--root'] ? argumentsParsed['--root'][0] : process.cwd(),
+    exclusions: argumentsParsed['--exclusions'] ? argumentsParsed['--exclusions'].map((str) => new RegExp(str)) : []
```

Comment:
```suggestion
    exclusions: argumentsParsed['--exclusions']?.map((str) => new RegExp(str)) ?? []
```

Reply:
I tried that too, but it just doesn´t get past the linter and I don´t understand why...

Reply:
nvm